test/integration/generated_runner.dart

```diff
@@ -0,0 +1,31 @@
+@TestOn('vm')
+
+/************ GENERATED FILE ************
+
+This file was generated with the command:
+gen-test-runner -d test/integration -e Environment.vm --no-react --no-genHtml 
```

Comment:
This would actually be `dart_dev gen-test-runner` or `pub run dart_dev gen-test-runner`, right?